capabilities-db/buildDB.sh

```diff
@@ -1,12 +1,33 @@
 #!/bin/bash
+
+# This script is meant to be run in the directory it's in.
+# Ensure the script has the correct running permissions. If not do: chmod +x buildDB.sh
+# Use: ./buildDB.sh <your_mysql_root_password>
+
 echo "Starting Script..."
 echo ""
 
+if [ $# -eq 0 ]
+  then
+    echo "No password to root supplied."
+    echo "./buildDB.sh <your_mysql_root_password>"
+    echo "If root has no password set use 'none' for the password"
+    exit
+fi
+
+PASS=$1
+
+if [ $PASS = "none" ]
+  then
+    echo "Using empty password."
+    PASS=""
+fi
+
 echo "Sourcing capabilitiesDB-setup_prod.sql..."
-mysql -u root -e "source capabilities-db/capabilitiesDB-setup_prod.sql;"
+mysql -u root --password="$PASS" -e "source capabilitiesDB-setup_prod.sql;"
```

Comment:
I feel like we shouldn't be typing the password in the script, since it would show up in bash history. Might be a good idea to cat the password from the mysql.env instead?

Reply:
That's a good point. I'll fix that.